Composer/packages/client/src/pages/publish/Publish.tsx

```diff
@@ -86,6 +86,19 @@ const Publish: React.FC<RouteComponentProps<{ projectId: string; targetName?: st
     [projectId, publishTypes]
   );
 
+  const isPublishDisable = useMemo(
+    () => () => {
+      // if current profile is already publishing, set disable to true
+      if (selectedTargetName && selectedTargetName !== 'all') {
+        if (thisPublishHistory?.length >= 0) {
+          return thisPublishHistory[0]?.status === 202;
+        }
+      } else {
+        return selectedTargetName === 'all';
+      }
+    },
```

Comment:
Why does this need to return a function?

Reply:
fixed